HW_6/HW_7.py

```diff
@@ -0,0 +1,175 @@
+
+import datetime
+import random
+import datefinder
+import HW_3_updated as normalization
+import sys, os
+import csv
+import re
+from collections import Counter
+
+sys.path.append('C:\\Users\\Kateryna_Kulmatytska\\PycharmProjects\\Python_HW\\HW_6')
+
+class Article:
+    def __init__(self, header, text):
+        self.header = header
+        self.text = text
+
+    def print_to_file(self, formatted_text, file_name):
+        with open(file_name, "a") as file:
+            file.write(formatted_text)
+            with open(file_name, "r") as file_updated:
+                content = file_updated.read()
+                words = re.findall(r"[\w']+", content.lower())
+                d = dict()
+                for word in words:
+                    if word.isalpha():
+                        if word in d:
+                            d[word] = d[word] + 1
```

Comment:
Or simplier:
```d[word] += 1```

Also, you could take a look on the Counter from the module collection, it can simplify the task of counting:

https://pythonworld.ru/moduli/modul-collections.html